src/Item.cpp

```diff
@@ -0,0 +1,53 @@
+//
+// Created by Jahrr on 4/6/2023.
+//
+// Last edited by Jahrr on 4/6/2023
+//
+#include "Item.hpp"
+
+// Number values that are not set are set to -1 by default
+Item::Item() : name_{"Not Set"}, item_ID_{-1}, price_{-1}, stock_{-1} {}
+
+Item::Item(int item_ID, std::string item_name, float item_price, int item_stock)
+    : name_{std::move(item_name)}, price_{item_price}, stock_{item_stock} {}
```

Comment:
item_ID_{item_ID};

Reply:
I think there should only be one constructor, and all the parameters should have optional values, because that allows for greater flexibility, and std::string_view should be used. Also the member variables in the initializer list are 
initialized in the order in which they are declared in the class, so they should match that order.
Here the explicit keyword is used to avoid implicit conversions.
`
explicit Item(std::string_view name = "", int item_ID = 0, float price = 0.0, int stock = 0)
        : name_ {name}, item_ID_ {item_ID}, price_ {price}, stock_ {stock} {}
        `

Reply:
String_view is now used, but the two constructors were kept to allow ease of object creation.